pyanaconda/ui/gui/spokes/software_selection.py

```diff
@@ -194,6 +226,14 @@ def apply(self):
         for group_name in self._get_selected_addons():
             self._selection.groups.append(group_name)
 
+        # Select kernel.
+        property_redhat_kernel = self.check_button_redhat_kernel.get_active()
+        property_64k = self.check_button_64k.get_active()
+        kernel = get_kernel_from_properties(property_redhat_kernel, property_64k)
+        if kernel is not None:
+            log.debug("Selected kernel package: %s", kernel)
+            self._selection.kernel_packages.append(kernel)
```

Comment:
Btw. I don't see code of the `refresh` method that would update the value of checkboxes based on the current packages selection.